src/FINDOLOGIC/Export/XML/XMLItem.php

```diff
@@ -122,4 +124,22 @@ private function buildUsergroups(\DOMDocument $document)
 
         return $usergroups;
     }
+
+    private function validateImages(array $images){
```

Comment:
A doc block would be nice to document parameters and describe what validity actually means.

Reply:
done